testing/test_mark.py

```diff
@@ -477,6 +516,19 @@ def test_some(request):
         reprec = testdir.inline_run("-m", "mark1")
         reprec.assertoutcome(passed=1)
 
+    def assert_markers(self, items, **expected):
```

Comment:
Should we move this to somewhere in `testdir`, or it is of little use outside of this module?